src/ensembl/src/shared/types/status.ts

```diff
@@ -0,0 +1,16 @@
+/*
+  This file contains the Status enumerable type.
+  The Status type does not correspond to any single thing,
+  but rather serves as a collection of various possible statuses
+  that various things may have.
+*/
+
+// To developer: for easier checking of which statuses are available,
+// please add new variants to the enum in the alphabetical order
+export enum Status {
+  ACTIVE = 'active',
+  DISABLED = 'disabled',
+  DEFAULT = 'default',
+  HIGHLIGHTED = 'highlighted',
+  INACTIVE = 'inactive'
+}
```

Comment:
Do you think we should build the `LoadingState` from this enum or keep that as a separate enum on it's own?

Reply:
I don’t know, to be honest. I am reluctant to put unrelated things in one place — it feels disorganized somehow. I would probably leave the loading state where it is.